packages/neos-ui/src/Containers/RightSideBar/Inspector/SelectedElement/style.css

```diff
@@ -0,0 +1,20 @@
+.label {
+    width: 100%;
+    overflow-x: hidden;
+    text-overflow: ellipsis;
+    padding: 0 40px 0 16px;
+    height: 40px;
+    margin: 0;
+    line-height: 40px;
+    font-weight: bold;
+    font-size: 14px;
```

Comment:
Also would be great if we eventually gather all font-sizes as constants too... not necessarily now.
This would be especially important when we would start working on the design of the UI again.